refbox/src/audienceInfo.js

```diff
@@ -18,7 +18,7 @@ function AudienceHeader(props) {
 
 function ChallengeDescription(props) {
   const defaultDescription = 'Here a description of the challenge should be displayed';
-  const description = props.info.description ? props.info.description : defaultDescription;
+  const description = props.description ? props.description : defaultDescription;
```

Comment:
Was this a typo? I can't find a property `info` in the diffs

Reply:
See line 271 (old) 274 (new). It currently works as planned

Reply:
See line 271 (old) 274 (new). It currently works as planned